server/capture.c

```diff
@@ -425,6 +426,7 @@ static void ensure_sample_count(
 
     /* Not being captured.  We need to emit a capture entry right now.  Because
      * we're first and only, we go into the anonymous group. */
+    capture->sample_count_anonymous = true;
```

Comment:
You're setting this here ... but where is it being reset?  Don't think it is...

Reply:
I'm relying on it being initialized to false when the structure is created. Does that work?

Reply:
ahh, I see it persists...